### ISSUE TITLE
Pod update

### ISSUE THREAD

**Issue by leszekkrol** [Baseflow/flutter-geolocator#659]
Hello 
After updating to the latest version, I have the following error:

[!] No podspec found for `geolocator_web` in `.symlinks/plugins/geolocator_web/ios`

**Comment by mvanbeusekom** [comment]
Could you provide a bit more details on what your are doing exactly (are you trying to run the App using `flutter run`, are you opening in Xcode, ...)? Could you also include the output of the `flutter doctor -v` command? 

This could be an issue if you are on an older Flutter version (pre 1.20).

**Comment by leszekkrol** [comment]
1. Update libraries in podspec
2. pod update cause this issue

**Comment by leszekkrol** [comment]
`
iMac:ios uogforceone$ pod update
Update all pods
Updating local specs repositories
Analyzing dependencies
[!] No podspec found for `geolocator_web` in `.symlinks/plugins/geolocator_web/ios`
iMac:ios uogforceone$ 
`

**Comment by mvanbeusekom** [comment]
Clear, thank you for providing this additional information. 

I will look into this tomorrow morning (it's kinda late here already).

**Comment by mvanbeusekom** [comment]
Maybe you can verify one more thing, do you get the same if you create a new Flutter app and add the geolocator plugin as a dependency?

**Comment by leszekkrol** [comment]
I've checked twice :) Still I got this issue.

**Comment by adnankhan74444** [comment]
I am also facing this issue. 
[!] No podspec found for `geolocator_web` in `.symlinks/plugins/geolocator_web/ios`

**Comment by mvanbeusekom** [comment]
I am having problems to reproduce the issue, here the steps I am taking:

1. Create a new Flutter project: `flutter create geo_issue_659`;
2. Change into the new project directory: `cd geo_issue_659`;
3. Edit the `pubspec.yaml` file to add a dependency on the geolocator (@leszekkrol note that I use version 6.2.0, in your issue you mentioned 6.1.20 which I assume is just a typo): 
```yaml
dependencies:
  flutter:
    sdk: flutter

  geolocator: ^6.2.0
```
4. Run `flutter pub get`;
5. Change into the iOS directory: `cd ios`;
6. Run the pod update command: `pod update`;

I am not running into any issues here, the output of the `pod update` command is as follows:

<details>
<summary>pod update</summary>

```bash
Update all pods
Updating local specs repositories
Analyzing dependencies
Downloading dependencies
Installing Flutter (1.0.0)
Installing geolocator (6.2.0)
Generating Pods project
Integrating client project
Pod installation complete! There are 2 dependencies from the Podfile and 2 total pods installed.

[!] Automatically assigning platform `iOS` with version `9.0` on target `Runner` because no platform was specified. Please specify a platform for this target in your Podfile. See `https://guides.cocoapods.org/syntax/podfile.html#platform`.

[!] CocoaPods did not set the base configuration of your project because your project already has a custom config set. In order for CocoaPods integration to work at all, please either set the base configurations of the target `Runner` to `Target Support Files/Pods-Runner/Pods-Runner.profile.xcconfig` or include the `Target Support Files/Pods-Runner/Pods-Runner.profile.xcconfig` in your build configuration (`Flutter/Release.xcconfig`).
```

</details>

I have tested on Flutter Stable same:

<details>
<summary>flutter doctor -v</summary>

```bash
[✓] Flutter (Channel stable, 1.22.6, on macOS 11.2 20D64 darwin-x64, locale en-GB)
    • Flutter version 1.22.6 at /Users/maurits/fvm/versions/stable
    • Framework revision 9b2d32b605 (2 weeks ago), 2021-01-22 14:36:39 -0800
    • Engine revision 2f0af37152
    • Dart version 2.10.5

[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.2)
    • Android SDK at /Users/maurits/Library/Android/sdk
    • Platform android-30, build-tools 30.0.2
    • ANDROID_HOME = /Users/maurits/Library/Android/sdk
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6222593)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 12.4)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 12.4, Build version 12D4e
    • CocoaPods version 1.10.1

[!] Android Studio (version 4.1)
    • Android Studio at /Applications/Android Studio.app/Contents
    ✗ Flutter plugin not installed; this adds Flutter specific functionality.
    ✗ Dart plugin not installed; this adds Dart specific functionality.
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6222593)

[✓] IntelliJ IDEA Ultimate Edition (version 2020.3.1)
    • IntelliJ at /Applications/IntelliJ IDEA.app
    • Flutter plugin installed
    • Dart plugin version 203.6912

[✓] VS Code (version 1.53.0)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.19.0


[!] Connected device
    ! No devices available

! Doctor found issues in 2 categories.
```

</details>

Doing a quick google search on the error message resulted in the following [SO post](https://stackoverflow.com/questions/65230470/flutter-ios-no-podspec-found-for-flutter-keyboard-visibility-web-in/65244666), maybe this will help? 

TL;DR:
1. Delete Podfile and Podfile.lock from `ios` folder;
2. Run `flutter clean` in the root folder of your project;
3. Change back into the `ios` folder and run `pod install`;
4. All should be good and you should be able to run you App.

Let me know if this works.

**Comment by leszekkrol** [comment]
@matheusrezende Works fine! I am surprised now because I was following a similar list of steps and it did not work yesterday. Thanks!

**Comment by mvanbeusekom** [comment]
That is good to hear, thank you for testing and reporting back. This will also be useful for other users experiencing a similar issue. 

I will close this issue now, feel free to leave a comment if you run into additional problems.

**Comment by adnankhan74444** [comment]
i am still facing the same issue please guide how to solve this problem ?

**Comment by adnankhan74444** [comment]
pod update
Update all pods
Updating local specs repositories
Analyzing dependencies
[!] No podspec found for `geolocator_web` in `.symlinks/plugins/geolocator_web/ios`

**Comment by adnankhan74444** [comment]
Launching lib/main.dart on iPhone 8 in debug mode...
Running pod install...
CocoaPods' output:
↳
      Preparing

    Analyzing dependencies

    Inspecting targets to integrate
      Using `ARCHS` setting to build architectures of target `Pods-Runner`: (``)

    Fetching external sources
    -> Fetching podspec for `Flutter` from `Flutter`
    -> Fetching podspec for `barcode_scan` from `.symlinks/plugins/barcode_scan/ios`
    -> Fetching podspec for `camera` from `.symlinks/plugins/camera/ios`
    -> Fetching podspec for `connectivity` from `.symlinks/plugins/connectivity/ios`
    -> Fetching podspec for `connectivity_for_web` from `.symlinks/plugins/connectivity_for_web/ios`
    -> Fetching podspec for `connectivity_macos` from `.symlinks/plugins/connectivity_macos/ios`
    -> Fetching podspec for `device_id` from `.symlinks/plugins/device_id/ios`
    -> Fetching podspec for `device_info` from `.symlinks/plugins/device_info/ios`
    -> Fetching podspec for `firebase_analytics` from `.symlinks/plugins/firebase_analytics/ios`
    -> Fetching podspec for `firebase_core` from `.symlinks/plugins/firebase_core/ios`
    -> Fetching podspec for `firebase_core_web` from `.symlinks/plugins/firebase_core_web/ios`
    -> Fetching podspec for `firebase_messaging` from `.symlinks/plugins/firebase_messaging/ios`
    -> Fetching podspec for `flutter_exif_rotation` from `.symlinks/plugins/flutter_exif_rotation/ios`
    -> Fetching podspec for `flutter_facebook_login` from `.symlinks/plugins/flutter_facebook_login/ios`
    -> Fetching podspec for `flutter_image_compress` from `.symlinks/plugins/flutter_image_compress/ios`
    -> Fetching podspec for `flutter_inappwebview` from `.symlinks/plugins/flutter_inappwebview/ios`
    -> Fetching podspec for `flutter_local_notifications` from `.symlinks/plugins/flutter_local_notifications/ios`
    -> Fetching podspec for `flutter_plugin_android_lifecycle` from `.symlinks/plugins/flutter_plugin_android_lifecycle/ios`
    -> Fetching podspec for `flutter_statusbarcolor` from `.symlinks/plugins/flutter_statusbarcolor/ios`
    -> Fetching podspec for `flutter_uploader` from `.symlinks/plugins/flutter_uploader/ios`
    -> Fetching podspec for `geolocator` from `.symlinks/plugins/geolocator/ios`
    -> Fetching podspec for `geolocator_web` from `.symlinks/plugins/geolocator_web/ios`
    [!] No podspec found for `geolocator_web` in `.symlinks/plugins/geolocator_web/ios`

    /Library/Ruby/Gems/2.6.0/gems/cocoapods-1.10.1/lib/cocoapods/external_sources/path_source.rb:14:in `block in fetch'
    /Library/Ruby/Gems/2.6.0/gems/cocoapods-1.10.1/lib/cocoapods/user_interface.rb:64:in `section'
    /Library/Ruby/Gems/2.6.0/gems/cocoapods-1.10.1/lib/cocoapods/external_sources/path_source.rb:11:in `fetch'
    /Library/Ruby/Gems/2.6.0/gems/cocoapods-1.10.1/lib/cocoapods/installer/analyzer.rb:989:in `fetch_external_source'
    /Library/Ruby/Gems/2.6.0/gems/cocoapods-1.10.1/lib/cocoapods/installer/analyzer.rb:968:in `block (2 levels) in fetch_external_sources'
    /Library/Ruby/Gems/2.6.0/gems/cocoapods-1.10.1/lib/cocoapods/installer/analyzer.rb:967:in `each'
    /Library/Ruby/Gems/2.6.0/gems/cocoapods-1.10.1/lib/cocoapods/installer/analyzer.rb:967:in `block in fetch_external_sources'
    /Library/Ruby/Gems/2.6.0/gems/cocoapods-1.10.1/lib/cocoapods/user_interface.rb:64:in `section'
    /Library/Ruby/Gems/2.6.0/gems/cocoapods-1.10.1/lib/cocoapods/installer/analyzer.rb:966:in `fetch_external_sources'
    /Library/Ruby/Gems/2.6.0/gems/cocoapods-1.10.1/lib/cocoapods/installer/analyzer.rb:117:in `analyze'
    /Library/Ruby/Gems/2.6.0/gems/cocoapods-1.10.1/lib/cocoapods/installer.rb:414:in `analyze'
    /Library/Ruby/Gems/2.6.0/gems/cocoapods-1.10.1/lib/cocoapods/installer.rb:239:in `block in resolve_dependencies'
    /Library/Ruby/Gems/2.6.0/gems/cocoapods-1.10.1/lib/cocoapods/user_interface.rb:64:in `section'
    /Library/Ruby/Gems/2.6.0/gems/cocoapods-1.10.1/lib/cocoapods/installer.rb:238:in `resolve_dependencies'
    /Library/Ruby/Gems/2.6.0/gems/cocoapods-1.10.1/lib/cocoapods/installer.rb:160:in `install!'
    /Library/Ruby/Gems/2.6.0/gems/cocoapods-1.10.1/lib/cocoapods/command/install.rb:52:in `run'
    /Library/Ruby/Gems/2.6.0/gems/claide-1.0.3/lib/claide/command.rb:334:in `run'
    /Library/Ruby/Gems/2.6.0/gems/cocoapods-1.10.1/lib/cocoapods/command.rb:52:in `run'
    /Library/Ruby/Gems/2.6.0/gems/cocoapods-1.10.1/bin/pod:55:in `<top (required)>'
    /usr/local/bin/pod:23:in `load'
    /usr/local/bin/pod:23:in `<main>'

Error running pod install
Error launching application on iPhone 8.

**Comment by manfye** [comment]
still the same issue here

**Comment by adnankhan74444** [comment]
any update on this issue ?

**Comment by mvanbeusekom** [comment]
@adnankhan74444 what have you tried?

Please check the Flutter version you are using (should be 1.20 or higher). Next make sure to follow these steps:

1. Delete `Podfile` and `Podfile.lock` from `ios` folder;
2. Run `flutter clean` in the root folder of your project;
3. Change back into the `ios` folder and run `pod install`;
4. All should be good and you should be able to run you App.

**Comment by puhgeh** [comment]
> any update on this issue ?

@adnankhan74444 
These steps worked for me:
1. Delete `Podfile` and `Podfile.lock`
2. Run `flutter clean` in the root folder of project.
3. Run `flutter pub get`
4. Open `Podfile` and specify the minimum iOS version (uncomment the line with the iOS version; change if necessary)
5. Click the `run` button on your IDE to build and run your App.

**Comment by nickrivers** [comment]
I have the same problem that @adnankhan74444 

Any update of this?

I'm trying the @mvanbeusekom solution but doesn't works

**Comment by lizhuoyuan** [comment]
same issue
version: 6.2.1
flutter: 1.22.6
